File: SOC/src/cc/game/soc/core/BotNodeDice.java
package cc.game.soc.core;

public class BotNodeDice extends BotNode {

	private Integer [] dice = new Integer[2];
	
	public BotNodeDice() {}

	BotNodeDice(int die0, int die1) {
		dice[0] = die0;
		dice[1] = die1;
	}
	
	@Override
	public Object getData() {
		return dice;
	}

	@Override
	public String getDescription() {
		return "Dice [" + dice[0] + " " + dice[1] + "]";
	}
	
	
}

File: SOC/src/cc/game/soc/core/BotNodeDice.kt
package cc.game.soc.core

import cc.game.soc.core.BotNode

class BotNodeDice(die0: Int, die1: Int) : BotNode() {
	private val dice = arrayOf(die0, die1)

	override val data = dice

	override val description: String = "Dice [${dice[0]} ${dice[1]}]"
}